test/test.js

```diff
@@ -42,15 +47,16 @@ test('all modules', function (t) {
   t.deepEqual(hook.cache['http'], http)
   t.deepEqual(hook.cache['net'], net)
   t.equal(n, 3)
+
+  hook.unhook()
```

Comment:
Can you do these in a `t.on('end', () => { ... })` immediately after the hook is created so they still run if something throws before they are reached?

Reply:
Ah nice! I didn't know `t` was an EventEmitter. Updated 😄